packages/popover2/src/contextMenu2.tsx

```diff
@@ -227,22 +229,22 @@ export const ContextMenu2: React.FC<ContextMenu2Props> = ({
             contentProps,
             onContextMenu: handleContextMenu,
             popover: maybePopover,
-            ref: containerRef,
+            ref: mergeRefs(containerRef, userRef),
```

Comment:
we probably don't need this. the user has the ref already so we don't need to send it back to them in the child render callback

Reply:
```suggestion
            ref: containerRef,
```